src/pages/faq.jsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom"; // Import useNavigate
+import faqData from "../data/morefaq.json";
+
+function FAQPage() {
+  const [openQuestion, setOpenQuestion] = useState(null);
+  const navigate = useNavigate(); // Initialize useNavigate
+
+  const toggleQuestion = (index) => {
+    setOpenQuestion(openQuestion === index ? null : index);
+  };
+
+  const goToHome = () => {
+    navigate("/"); // Navigate to the home route
+  };
+
+  return (
+    <section className="bg-gradient-to-br from-gray-900 via-gray-800 to-gray-900 py-12">
+      <div className="mx-auto w-full max-w-7xl px-4"> {/* Changed px-45 to px-4 for Tailwind */}
+        <h2 className="text-5xl font-bold text-white mb-8 text-center">All FAQs</h2>
```

Comment:
I believe we already have FAQ component you can use that in both places.
One home page use prop to check how many faq to show by default show all.